src/core/prompts/__tests__/__snapshots__/system.test.ts.snap

```diff
@@ -662,50 +662,46 @@ Example: Insert a new function and its import statement
 Description: Request to perform search and replace operations on a file. Each operation can specify a search pattern (string or regex) and replacement text, with optional line range restrictions and regex flags. Shows a diff preview before applying changes.
 Parameters:
 - path: (required) The path of the file to modify (relative to the current workspace directory /test/path)
-- operations: (required) A JSON array of search/replace operations. Each operation is an object with:
-    * search: (required) The text or pattern to search for
-    * replace: (required) The text to replace matches with. If multiple lines need to be replaced, use "
-" for newlines
-    * start_line: (optional) Starting line number for restricted replacement
-    * end_line: (optional) Ending line number for restricted replacement
-    * use_regex: (optional) Whether to treat search as a regex pattern
-    * ignore_case: (optional) Whether to ignore case when matching
-    * regex_flags: (optional) Additional regex flags when use_regex is true
-Usage:
-<search_and_replace>
-<path>File path here</path>
-<operations>[
-  {
-    "search": "text to find",
-    "replace": "replacement text",
-    "start_line": 1,
-    "end_line": 10
-  }
-]</operations>
-</search_and_replace>
-Example: Replace "foo" with "bar" in lines 1-10 of example.ts
+- operations: (required) One or more search/replace operations in the following format:
```

Comment:
Do you think it's important that this supports multiple operations? Seems like it might be simpler if it was just one search block and one replace block.

Reply:
@mrubens  i think we can do that for this search_and_replace and insert_content too ?

Reply:
Yeah agree